app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
```

Comment:
## Provide  sign in/signup and sign out links to the user
- [ ] Once we have implemented the logic for a user to log out, let's make ensure we have a Sign in/Sign out links present in any our views here ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/sign_in_out_link_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49722686/))